components/raftstore/src/store/fsm/store.rs

```diff
@@ -1009,6 +1009,7 @@ impl RaftBatchSystem {
         mut coprocessor_host: CoprocessorHost,
         importer: Arc<SSTImporter>,
         split_check_worker: Worker<SplitCheckTask>,
+        auto_split_controller: AutoSplitController,
```

Comment:
Can this parameter be reduced?

Reply:
I have no good idea. we need to register config in `server.rs` and start `stat_monitor` in `pd.rs`